### ISSUE TITLE
[Dockerimage] Can't upload large documents, 413 Request Entitiy too large

### ISSUE THREAD

**Issue by frenos** [ciur/papermerge#148]
**Bug Description**
If you try to upload a document with a lot of pages the upload silently fails. In the browser console you see the `/upload` POST got 413 Request Entity too large. The document I tried to upload is 10,8Mb, sadly I cant provide the file because of private data.

**Expected behavior**
The default value should be higher and documentation updated. It would also be great if the frontend tells you if something went wrong. For private installs with trusted users it would also be nice to have an option to set the max-size to 0 (as in disable a maximum)

**Desktop (please complete the following information):**
 - OS: your docker images
 - Browser: Firefox 81.0
 - Papermerge Version: 1.4.5

**Comment by ciur** [comment]
Hi @frenos !
Thank for opening an inssue. This bug has nothing to do with application code, it is because of webserver (application stays behind webserver) misconfiguration.
You use docker image right ?
If yes, which docker image do you use ? [is it the official one?](https://hub.docker.com/repository/docker/eugenci/papermerge)
In case you use docker image provided by linuxserver.io, [you need to open bug in their repo.](https://github.com/linuxserver/docker-papermerge).

**Comment by ciur** [comment]
modwsgi by default limits max http body size to 10MB. From modwsgi udocumentation:
```
 Option  --limit-request-body NUMBER
                        The maximum number of bytes which are allowed in a
                        request body. Defaults to 10485760 (10MB).
 ```
I will increase that value to 20 MB. Fix will be available in mid October (will be shipped with 1.5).

**Comment by ciur** [comment]
Fix is now available as part of [1.5.0.rc1](https://github.com/ciur/papermerge/releases/tag/v1.5.0.rc1)
